starttls_policy/tests/configure_test.py

```diff
@@ -86,5 +122,24 @@ def test_instruct_string(self):
         self.assertTrue("postfix reload" in instructions)
         self.assertTrue(generator.default_filename in instructions)
 
+def enclose_config_case(conf, expected):
+    """Wrap testcase against config into closure in order to make it attachable
+    to main testcase class"""
```

Comment:
Maybe: Returns parametrized test for transforming policy file into Postfix config